Library/Homebrew/cask/exceptions.rb

```diff
@@ -174,7 +174,7 @@ def initialize(path, reason)
     end
 
     def to_s
-      s = "Failed to quarantine #{path}."
+      s = +"Failed to quarantine #{path}."
```

Comment:
Would be good to freeze these all before return.

Reply:
Done in https://github.com/Homebrew/brew/pull/6075

Reply:
Thanks. 👍